runtime/service_integ_test.go

```diff
@@ -1546,6 +1547,35 @@ func TestStopVM_Isolated(t *testing.T) {
 				}
 			},
 		},
+
+		{
```

Comment:
Please add some high level indication on what this is testing.

Reply:
Done